packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -534,9 +533,25 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         address prevADDRESS = messageContext.ovmADDRESS;
         messageContext.ovmADDRESS = eoa;
 
-        // Now actually create the account and get its bytecode. We're not worried about reverts
-        // (other than out of gas, which we can't capture anyway) because this contract is trusted.
-        OVM_ProxyEOA proxyEOA = new OVM_ProxyEOA(0x4200000000000000000000000000000000000003);
+        // Creates a duplicate of the OVM_ProxyEOA located at 0x42....09. Uses the following
```

Comment:
Note that this prevents us from doing any interesting logic in the constructor of the proxy EOA. Just something to be careful about in the future. Other option is to use the initcode instead of the deployed bytecode but this is a bit harder to fit into our deployment flow.

Reply:
Added a comment to ProxyEOA to make sure people don't use constructor logic